### PR TITLE
Update python-wxgtk for ubuntu and debian

### PR DESCRIPTION
**Pull request by jackieokay**
This is a temporary measure to fix the failing robot_model job, because python-wxgtk2.8 is not available in Jessie or Xenial. The eventual goal is to replace the WxWidgets dependency in joint_state_publisher with Qt.

### CODE REVIEW

File: rosdep/base.yaml [hunk 1]
```diff
@@ -3644,14 +3644,32 @@ wx-common:
 wxpython:
   arch: [wxpython]
   centos: [wxPython-devel]
-  debian: [python-wxgtk2.8]
+  debian:
+    jessie: [python-wxgtk3.0]
+    sid: [python-wxgtk2.8]
```
Comment (tfoote) [inline rosdep/base.yaml]: Sid should be 3.0 it's the experimental rolling release for debian. https://packages.debian.org/sid/python-wxgtk3.0

**Comment by tfoote** [comment]
Ubuntu versions of 2.8: http://packages.ubuntu.com/search?keywords=python-wxgtk2.8 ends with wily
Ubuntu versions of 3.0: http://packages.ubuntu.com/search?keywords=python-wxgtk3.0

Debian version 2.8 ends with wheezy: https://packages.debian.org/wheezy/python-wxgtk2.8
Debian version for 3.0 starts with Jessie: https://packages.debian.org/sid/python-wxgtk3.0

**Comment by jacquelinekay** [comment]
thanks @tfoote, I'm still getting used to Debian... Does it look correct now?

**Comment by jacquelinekay** [comment]
Going to close this in favor of ros/robot_model#128.

**Comment by tfoote** [comment]
It may not be used for this use case but I think we've converged on the accurate rosdep rules. @jacquelinekay I think this is worth merging to keep correctness in the rules. Thoughts?

**Comment by jacquelinekay** [comment]
Sure, why not. I don't need it anymore but maybe someone else really wants wxWidgets.

**Pull request merged**